conda/_vendor/toolz/dicttoolz.py

```diff
@@ -1,6 +1,7 @@
 import copy
 import operator
-from .compatibility import map, zip, iteritems, iterkeys, itervalues, reduce
+from conda._vendor.toolz.compatibility import (map, zip, iteritems, iterkeys, itervalues,
+                                 reduce)
```

Comment:
I'm not sure why we need to make the import explicit here? Or was this rewritten by vendoring?

Reply:
vendoring magic